src/components/dialog/dialog-container.ts

```diff
@@ -0,0 +1,70 @@
+import {Component, ComponentRef, ViewChild, AfterViewInit} from '@angular/core';
+import {BasePortalHost, ComponentPortal, TemplatePortal} from '../../core/portal/portal';
```

Comment:
Is there a reason this one uses `../../core/portal/` rather than `@angular2-material/core/portal/` ?